scaling-puppet-enterprise.md

```diff
@@ -50,7 +50,11 @@ A rule of thumb for the PuppetDB service is that at most a single PuppetDB servi
 
 ### Alternate Options
 
-In order to mediate a known spike in JRuby use for pluginsync, you can install the [bulk_pluginsync module](https://forge.puppet.com/npwalker/bulk_pluginsync) which avoids the need to use pluginsync by having your agent installation script download a tarball of the plugins from your master and extract it into place before installing the agent.
+In order to mediate a known spike in JRuby use for pluginsync, you can use bulk_pluginsync which avoids the need to use pluginsync by having your agent installation script download a tarball of the plugins from your master and extract it into place before installing the agent. Bulk_pluginsync can be used in the following ways: 
```

Comment:
Nitpicky, but I wouldn't say this

> avoids the need to use pluginsync

It dramatically reduces (potentially to 0) the need for pluginsync on the initial puppet run of a new agent installation.  Pluginsync will still be used from then on, and it would still be used if there was a change to plugins on the master within the last hour (looks like the cronjob runs hourly).

Reply:
To nitpick even more, I wouldn't say this has any effect on the "need to use pluginsync". If you are using modules, you need to do plugin sync. Full stop.

What this does reduce is the amount of work that pluginsync needs to do.

Reply:
updated